buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/TestClustersPlugin.java

```diff
@@ -86,6 +102,10 @@ public void apply(Project project) {
 
             // After each task we determine if there are clusters that are no longer needed.
             configureStopClustersHook(project);
+
+            // Since we have everything modeled in the DSL, add all the required dependencies e.x. the distribution to the
```

Comment:
You are just configuring the dependencies based on the DSL here, right?

Reply:
That's right.